test/job_packager.test.js

```diff
@@ -1,174 +0,0 @@
-'use strict';
-
-describe('job packager', function(){
-  var JobPackager = require('../src/job_packager');
-  var errors = require('../src/errors');
```

Comment:
I changed the name to job_packager.test*s*.js. Then I merged dev to shared, that file thrown some conflicts and I forget to delete it.
I merge the conflicting changes into the renamed file before I delete it.